File: Android/Martinshare/app/src/main/java/mobile/martinshare/com/martinshare/API/protocols/GetEintraegeProtocol.java
package mobile.martinshare.com.martinshare.API.protocols;

/**
 * Created by Modestas Valauskas on 03.06.2015.
 */
public interface GetEintraegeProtocol {
    public void startedGetting();
    public void notLoggedIn();
    public void noInternet();
    public void notChanged(boolean warn);
    public void aktualisiert(boolean warn);
}

File: iOS/Martinshare iOS/Martinshare/GetEintraegeProtocol.swift
//
//  GetEintraegeProtocol.swift
//  Martinshare
//
//  Created by Modestas Valauskas on 29.05.15.
//  Copyright (c) 2015 ModestasV Studios. All rights reserved.
//

import Foundation

protocol GetEintraegeProtocol {

    func startedGetting()
    func notLoggedIn()
    func unknownError()
    func noInternet()
    func notChanged(warn: Bool)
    func aktualisiert(warn: Bool, afterAktualisiertOderAktuell: (()-> Void))
}